packages/js/email-editor/src/store/actions.ts

```diff
@@ -1,139 +1,31 @@
 /**
  * External dependencies
  */
-import { dispatch, select } from '@wordpress/data';
-import { store as interfaceStore } from '@wordpress/interface';
```

Comment:
Since we don't need `@wordpress/interface` anymore, can we remove these
* https://github.com/woocommerce/woocommerce/blob/173bae563b0e0741f3c1cb63760696a7715be2a1/packages/js/email-editor/src/types/index.ts#L7
* https://github.com/woocommerce/woocommerce/blob/173bae563b0e0741f3c1cb63760696a7715be2a1/packages/js/email-editor/src/types/index.ts#L110
* https://github.com/woocommerce/woocommerce/blob/173bae563b0e0741f3c1cb63760696a7715be2a1/packages/js/email-editor/src/types/wordpress-modules.ts#L48-L67